.github/workflows/go.yml

```diff
@@ -0,0 +1,130 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+      - release/*
+      # skip non-trunk PMM-XXXX/SAAS-XXXX branch builds, but still build pull requests
+      - '!PMM-[0-9]+'
+      - '!SAAS-[0-9]+'
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+
+jobs:
+  build:
+    name: Build
+
+    strategy:
+      fail-fast: false
+      matrix:
+        go:
+          - version: 1.16.x
+            may-fail: false
+          - version: tip
+            may-fail: true
+
+    continue-on-error: ${{ matrix.go.may-fail }}
+
+    runs-on: ubuntu-latest
+
+    env:
+      GOPATH: ${{ github.workspace }}
+      WORKING_DIR: ${{ github.workspace }}/src/github.com/percona/pmm
```

Comment:
We don't use it in other CI workflows. Why do we need it here? It's used in just one place.

Reply:
It's needed to run unit tests correctly.